homeassistant/components/atag/sensor.py

```diff
@@ -0,0 +1,43 @@
+"""Initialization of ATAG One sensor platform."""
+from . import DOMAIN, AtagEntity
+
+DEFAULT_SENSORS = [
```

Comment:
Rename this constant to `AVAILABLE_SENSORS`.
```suggestion
AVAILABLE_SENSORS= [
```

Reply:
replaced this with ENTITY_TYPES which also replaces the import from pypi dependency